solution/backend/resources/views/resources.py

```diff
@@ -79,10 +82,14 @@ def format_gov_results(self, gov_results):
     def get_gov_results(self, query, page):
         key = os.environ.get('SEARCHGOV_KEY')
         offset = (page - 1) * self.limit
-
         rstring = f'https://search.usa.gov/api/v2/search/?affiliate=reg-pilot-cms-test&access_key={key}' \
                   f'&query={urlparse.quote_plus(query)}&limit={self.limit}&offset={offset}'
-        gov_results = json.loads(requests.get(rstring).text)
+        try:
+            gov_results = json.loads(requests.get(rstring, timeout=3).text)
+        except ConnectTimeout:
+            raise NotFound("Errors.  Search.gov took to long to return a response.")
+        except ValueError:
```

Comment:
When search.gov is having too many request it doesnt return a json response but a web page that was causing a parser error.